src/lib/filters/FilterButtons.svelte

```diff
@@ -0,0 +1,72 @@
+<script>
+	import { goto } from '$app/navigation';
+
+	export let query;
+
+	let categoryFiltered;
+	let targetGroupFiltered;
+	let minimumPlayersFiltered;
+	let materialFiltered;
+
+	$: categoryFiltered = query.getAll('category');
+	$: targetGroupFiltered = query.getAll('targetGroup');
+	$: minimumPlayersFiltered = query.getAll('minimumPlayers');
+	$: materialFiltered = query.getAll('material');
+
+	const removeCategoryFilter = () => {
+		query.delete('category');
+		goto(`/spellen/?${query}`);
+		categoryFiltered.length = 0;
+	};
+
+	const removeTargetGroupFilter = () => {
+		query.delete('targetGroup');
+		goto(`/spellen/?${query}`);
+		targetGroupFiltered.length = 0;
+	};
+
+	const removeMinimumPlayerFilter = () => {
+		query.delete('minimumPlayers');
+		goto(`/spellen/?${query}`);
+		minimumPlayersFiltered.length = 0;
+	};
+
+	const removeMaterialFilter = () => {
+		query.delete('material');
+		goto(`/spellen/?${query}`);
+		materialFiltered.length = 0;
+	};
+</script>
+
+<div>
+	{#if categoryFiltered.length > 0}
+		<button on:click={removeCategoryFilter}>Spelsoort <span>-</span></button>
+	{/if}
+	{#if targetGroupFiltered.length > 0}
+		<button on:click={removeTargetGroupFilter}>Groepen <span>-</span></button>
+	{/if}
+	{#if minimumPlayersFiltered.length > 0}
+		<button on:click={removeMinimumPlayerFilter}>Minimum spelers <span>-</span></button>
+	{/if}
+	{#if materialFiltered.length > 0}
+		<button on:click={removeMaterialFilter}>Materialen <span>-</span></button>
+	{/if}
+</div>
+
+<style>
+	button {
+		padding: 0.3rem 0.7rem;
```

Comment:
Zie het wel aantal keer voorbij komen. Wellicht niet door jou, omdat je het puur gekopieerd hebt. Maar volgens mij zijn we niet echt meer consistent aan wat we eerder zeiden over die `em`

Reply:
REM en EM sluiten elkaar niet uit, beide hebben een use case.

Reply:
Dat is hoe we het hebben afgesproken